MostlyArmless/ChatApp
# Mike chatroom broadcast

When a client sends a chat message, the target chatroom now rebroadcasts that message to all members of that chatroom, including the author of that message.

To test this:

1. Run the server in debug
2. Connect to the server from the Smart Websocket Client chrome extension and send a dummy clientIntro message
3. `npm start` the frontend which will connect to the server.
4. Type a chat message in the react frontend and press enter to send it
expected: the message should show up in the console, and in the Smart Websocket Client's received message window.
